update/update-version.md

```diff
@@ -1,139 +1,187 @@
     ```
-    In the unlikely event that an issue occurs, you can undo the modifications made by the script by running it again with the -z option. For example,
-    ```sh
-    bash update_v3.x.x_to_v4.x.x.sh -z
-    ```
-    Reverting the Terraform state file back to its prior condition, which existed prior to the execution of the script.
 
-    **Note:** If the script did fail, once the user reverts, they should not proceed with remaining steps and instead create an IBM Cloud support case and reference the script along with the error.
+    - Replace `<vpc-id1>` and `<vpc-region>` with the information that you found earlier.
+
+        If the job fails, see [Reverting changes](#reverting-changes).
+
+1. Initialize, check the planned changes, and apply the changes:
+    1. Update the version of the module in your consuming code to the 4.x.x version, as in this example:
 
-1. Now pull in the latest release and run `terraform plan` and make sure none of the VSIs will be recreated.
-    1. Update the version of the module in your consuming code to the 4.x.x version. For example:
         ```hcl
         source                           = "terraform-ibm-modules/landing-zone-vsi/ibm"
         version                          = "4.0.0"
         ```
-    1. Run `terraform init` to pull the latest version
-    1. Run `terraform plan` to ensure none of the VSIs will be recreated. You should see an update in-place for all of the VSIs in your state. The update will rename the VSI to the new naming convention of `prefix + last 4 digits of subnet_id + padding per subnet`. For example:
-        ```sh
-        ~ name                             = "prefix-001" -> "prefix-3ad7-001"
-        ```
 
-1. Run `terraform apply` to upgrade to the 4.x.x version of the module and apply the update in place to rename the VSIs.
+    1. Run the `terraform init` command to pull the latest version.
+    1. Run the `terraform plan` command to make sure that none of the VSIs will be re-created.
+
+        - You should see changes only to names. No resources should be set to be destroyed or re-created.
+        - The name changes include a prefix change: `prefix-` + the last 4 digits of the subnet ID + a sequential number for each subnet`.
+
+            For example,
+
+            ```sh
+            ~ name                             = "prefix-001" -> "prefix-3ad7-001"
+            ```
+
+    1. Run `terraform apply` to upgrade to the 4.x.x version of the module and apply the update in place to rename the VSIs.
+    1. If the commands are successful, follow the steps in [Clean up](#clean-up).
+
+### Clean up
+
+After you upgrade to the newer release of the VSI module, you can remove the temporary files that are generated by the script by running this command:
 
-1. After the successful upgrade to the newer release of the VSI module, it is safe to remove the temporary files generated by the script..
     ```sh
```

Comment:
this is not rendering correctly:
![image](https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone-vsi/assets/29608224/7d5cb599-1b88-4b31-9e98-4250697bc821)